canvas/config.py

```diff
@@ -130,6 +130,14 @@ def get_argument_parser(description = '', course = True, **kwargs):
         action = 'store', type = str, default = None,
         help = 'A Canvas authentication token.')
 
+    parser.add_argument('--json',action = 'store_true',
+        help = 'Output in JSON format instead of TSV')
```

Comment:
If `--json` lives here, then `--table` should probably also live here.
They should also probably be mutually exclusive:
https://docs.python.org/3/library/argparse.html#mutual-exclusion